---
video_input: 0              # Input Must be OpenCV readable 
visualize: True             # Disable for performance increase
vis_text: True              # Display fps on visualization stream
max_frames: 500             # only used if visualize==False
width: 300                  # 300x300 is used by SSD_Mobilenet -> highest fps
height: 300
fps_interval: 3             # Intervall [s] to print fps in console
det_interval: 75            # intervall [frames] to print detections to console
det_th: 0.5                 # detection threshold for det_intervall
split_model: True           # Splits Model into a GPU and CPU session (currently only works for ssd_mobilenets)
log_device: True            # Logs GPU / CPU device placement
allow_memory_growth: True   # limits memory allocation to the actual needs

# Change only if you use another model
model_name: 'ssd_mobilenet_v11_coco'
model_path: 'models/ssd_mobilenet_v11_coco/frozen_inference_graph.pb'
label_path: 'object_detection/data/mscoco_label_map.pbtxt'
num_classes: 90

#model_name: 'frcnni_handsnet_116_ss'
#model_path: 'models/frcnni_handsnet_116_ss/frozen_inference_graph.pb'
#label_path: 'models/frcnni_handsnet_116_ss/label_map.pbtxt'
#num_classes: 1
